src/docs/architecture.mdx

```diff
@@ -0,0 +1,18 @@
+[← Back to Documentation](/docs)
+
+### Query Connector ERD
+
+![Query Connector Database Structure](/Query-Connector-ERD.png)
```

Comment:
Heads up that this will break the image rendering in GitHub, though it still works properly on the site. That was the reason I added the remark plugin, but if it's causing slowdowns, it's not a huge deal if people can't see the images on GitHub.

Reply:
yeah basically we can't use turbopack properly if the plugin is here, so we can revisit adding it back if there's a need on GitHub